VT build fails to compile on Windows

https://openj9-jenkins.osuosl.org/job/Build_JDKnext_x86-64_windows_vt_standard_Personal/1/consoleFull

```
14:44:21  NativeHelpers.cpp
14:44:21  ObjectFieldInfo.cpp
14:44:22  [ 51%] Building CXX object runtime/vm/CMakeFiles/j9vm.dir/ObjectMonitor.cpp.obj
14:44:22  f:\users\jenkins\workspace\build_jdknext_x86-64_windows_vt_standard_personal\openj9\runtime\vm\ObjectFieldInfo.cpp(53): error C2220: the following warning is treated as an error
14:44:22  f:\users\jenkins\workspace\build_jdknext_x86-64_windows_vt_standard_personal\openj9\runtime\vm\ObjectFieldInfo.cpp(53): warning C4244: 'initializing': conversion from 'UDATA' to 'U_32', possible loss of data
14:44:22  f:\users\jenkins\workspace\build_jdknext_x86-64_windows_vt_standard_personal\openj9\runtime\vm\ObjectFieldInfo.cpp(77): warning C4319: '~': zero extending 'U_32' to 'UDATA' of greater size
14:44:22  make[6]: *** [runtime/vm/CMakeFiles/j9vm.dir/build.make:1618: runtime/vm/CMakeFiles/j9vm.dir/ObjectFieldInfo.cpp.obj] Error 2
14:44:22  make[6]: *** Waiting for unfinished jobs....
14:44:22  ObjectMonitor.cpp
```

--- a/runtime/vm/ObjectFieldInfo.cpp
+++ b/runtime/vm/ObjectFieldInfo.cpp
@@ -50,7 +50,7 @@ ObjectFieldInfo::countInstanceFields(void)
 				U_8 *fieldSigBytes = J9UTF8_DATA(J9ROMFIELDSHAPE_SIGNATURE(field));
 				if ('Q' == *fieldSigBytes) {
 					J9Class *fieldClass = findJ9ClassInFlattenedClassCache(_flattenedClassCache, fieldSigBytes + 1, J9UTF8_LENGTH(fieldSig) - 2);
-					U_32 size = fieldClass->totalInstanceSize;
+					U_32 size = (U_32)fieldClass->totalInstanceSize;
 					if (!J9_IS_FIELD_FLATTENED(fieldClass, field)) {
 						_instanceObjectCount += 1;
 						_totalObjectCount += 1;
@@ -74,7 +74,7 @@ ObjectFieldInfo::countInstanceFields(void)
 							}
 							_totalFlatFieldDoubleBytes += (U_32) ROUND_UP_TO_POWEROF2(size, sizeof(U_64));
 						} else if (J9_ARE_ALL_BITS_SET(fieldClass->classFlags, J9ClassLargestAlignmentConstraintReference)) {
-							size = (U_32) ROUND_UP_TO_POWEROF2(size, _referenceSize);
+							size = (U_32) ROUND_UP_TO_POWEROF2(size, (UDATA)_referenceSize);
 							_totalFlatFieldRefBytes += size;
 							setPotentialFlatObjectInstanceBackfill(size);
 						} else {

--- a/runtime/vm/resolvefield.cpp
+++ b/runtime/vm/resolvefield.cpp
@@ -1110,7 +1110,7 @@ fieldOffsetsFindNext(J9ROMFieldOffsetWalkState *state, J9ROMFieldShape *field)
 									state->objectsSeen++;
 								}
 							} else {
-								U_32 size = fieldClass->totalInstanceSize;
+								U_32 size = (U_32)fieldClass->totalInstanceSize;
 								bool forceDoubleAlignment = false;
 								if (sizeof(U_32) == referenceSize) {
 									/** 
@@ -1134,7 +1134,7 @@ fieldOffsetsFindNext(J9ROMFieldOffsetWalkState *state, J9ROMFieldShape *field)
 									Assert_VM_true((state->result.offset + referenceSize) % sizeof(U_64) == 0);
 									state->currentFlatDoubleOffset += ROUND_UP_TO_POWEROF2(size, sizeof(U_64));
 								} else if (J9_ARE_ALL_BITS_SET(fieldClass->classFlags, J9ClassLargestAlignmentConstraintReference)) {
-									size = ROUND_UP_TO_POWEROF2(size, referenceSize);
+									size = (U_32)ROUND_UP_TO_POWEROF2(size, referenceSize);
 									if (J9_ARE_ALL_BITS_SET(state->walkFlags, J9VM_FIELD_OFFSET_WALK_BACKFILL_FLAT_OBJECT_FIELD)
 										&& (state->flatBackFillSize == size)
 									) {

